libs/components/layout/src/lib/modules/back-to-top/back-to-top.directive.ts

```diff
@@ -62,16 +66,17 @@ export class SkyBackToTopDirective implements AfterViewInit, OnDestroy {
   constructor(
     private dockService: SkyDockService,
     private domAdapter: SkyBackToTopDomAdapterService,
-    private element: ElementRef
+    private element: ElementRef,
+    private scrollableHostService: SkyScrollableHostService
```

Comment:
I've modified things. I was attempting to work within the bounds of the existing public method on the adapter service. However, I went ahead and modified it to only take in an elementRef and then do the service call internally. I think we are ok there since the adapter isn't publicly exported through our index file.